Anytype/Sources/ServiceLayer/Object/TypesService/TypesService.swift

```diff
@@ -22,48 +28,55 @@ public final class TypesService: TypesServiceProtocol {
             $0.spaceID = spaceId
         }).invoke()
         
-        return try ObjectDetails(protobufStruct: result.details)
+        let objectDetails = try ObjectDetails(protobufStruct: result.details)
+        return ObjectType(details: objectDetails)
     }
     
     // MARK: - Search
-    public func searchObjectTypes(
+    func searchObjectTypes(
         text: String,
-        filteringTypeId: String? = nil,
-        shouldIncludeLists: Bool,
-        shouldIncludeBookmark: Bool,
+        includePins: Bool,
+        includeLists: Bool,
+        includeBookmark: Bool,
         spaceId: String
     ) async throws -> [ObjectDetails] {
```

Comment:
Maybe return `[ObjectType]`?

Reply:
Impossible ATM because of legacy screens. I will change it when we will remove toggle for new object search 👍